src/lib/action.rs

```diff
@@ -59,6 +59,10 @@ pub enum ActionType {
     MOVE,
     /// Update the text or label associated with a node.
     UPDATE,
+    /// Copy the text or label associated with a node or subtree.
+    COPY,
+    /// GLUE takes a subtree/node from tree1 and inserts it into tree2
```

Comment:
OK, but maybe say `from_node` and `parent` rather than `tree1` and `tree2` (to fit with your code in the struct)